tests/test_pytest_writer.py

```diff
@@ -1,51 +1,10 @@
 """Tests for behave_writer module."""
 import pathlib
-from types import SimpleNamespace
 
 import pytest
 
 from sphinx_gherkindoc import writer
-import rst_output_pytest as rst_output
-
-
-def _reformat_keywords(rst_lines):
-    new_rst_lines = rst_lines[:]
-    for (no_description_keyword, marker) in (("Background", "-"), ("Examples", "~")):
-        line_to_match = (
-            f":gherkin-{no_description_keyword.lower()}-keyword:"
-            f"`{no_description_keyword}:`"
-        )
-        for i, line in enumerate(new_rst_lines):
-            if line.startswith(line_to_match):
-                new_rst_lines[i] = f"{line_to_match}\n"
-            if new_rst_lines[i - 1] == f"{line_to_match}\n":
-                marker_line = marker * len(line_to_match)
-                new_rst_lines[i] = f"{marker_line}\n\n"
-    return new_rst_lines
-
-
-@pytest.fixture()
-def pytest_rst_output():
-    """Adjust expected rst output to align with pytest-bdd parser formats.
-
-    One change is necessary between ``behave`` and ``pytest-bdd`` formats.
-    Backgrounds and Examples in pytest-bdd do not have descriptions/titles,
-    so "Background: Stuff I Need" will become simply "Background:", for example.
-    As part of this change, the section header underline will also be adjusted
-    to match length.
-
-    This fixture creates a copy of the ``behave`` expected rst output module,
-    with that change made.
-    """
-    base_rst_output = SimpleNamespace()
-    for attribute in dir(rst_output):
-        if not attribute.startswith("_"):
-            setattr(
-                base_rst_output,
-                attribute,
-                _reformat_keywords(getattr(rst_output, attribute)),
-            )
-    return base_rst_output
```

Comment:
@bradsbrown I found it much easier to just be explicit in a file about what the rst will look like rather than have a partial copy that gets changed later. Made it much easier to update the current tests and add a new one.

I also had the thought that we need to make it easier to update these rst comparison tests. I usually end up catching the error and printing out the actual output, pasting it into the expected rst python file and then checking the diff to see what rst changed. It is really like the visual regression/snapshot idea where the tests should be able to automate the changes of the expected output automatically and then you just decide if you want to commit that.

Reply:
Yeah, I'm fine with that. My initial hope was to be able to circle back eventually to a place where both parsers returned exactly the same output for non-pytestbdd-specific things (allowing us to have common feature sources and expected rst outputs), so I had this function as a bit of a compatibility layer I hoped to be able to pull out eventually.

I'm pretty much resigned to that being a pipe dream at this point, so I'm fine stripping it out.